### PR TITLE
intalled google analytics

### PR DESCRIPTION
**Pull request by simonweniger**
intalled  `npm i react-ga -S` and added the tracking ID  from the new teal.garden property 
_(UA-147146482-3)_

google react plugin doesnt work...so i went with [this](https://www.gatsbyjs.org/packages/gatsby-plugin-google-analytics/) one providede by gatsby

Installed it:
`yarn add gatsby-plugin-google-analytics -W`

deleded the old one:
`yarn remove react-ga -W`

Added the details with our tracking ID to the` gatsby-config.js`

Also solved some spellings...

this one solves #21

### CODE REVIEW

File: packages/landing-gatsby/gatsby-config.js [hunk 1]
```diff
@@ -5,6 +5,33 @@ module.exports = {
...
+        // Avoids sending pageview hits from custom paths
+        exclude: ['/preview/**', '/do-not-track/me/too/'],
+        // Delays sending pageview hits on route update (in milliseconds)
+        pageTransitionDelay: 0,
+        // Enables Google Optimize using your container Id
+        optimizeId: 'YOUR_GOOGLE_OPTIMIZE_TRACKING_ID',
+        // Enables Google Optimize Experiment ID
+        experimentId: 'YOUR_GOOGLE_EXPERIMENT_ID',
+        // Set Variation ID. 0 for original 1,2,3....
+        variationId: 'YOUR_GOOGLE_OPTIMIZE_VARIATION_ID',
```
Comment (johba37) [inline packages/landing-gatsby/gatsby-config.js:28]: fill in or leave out?

File: packages/landing-gatsby/gatsby-config.js [hunk 2]
```diff
@@ -5,6 +5,33 @@ module.exports = {
...
+        // Any additional optional fields
+        sampleRate: 5,
+        siteSpeedSampleRate: 10,
+        cookieDomain: 'example.com',
```
Comment (johba37) [inline packages/landing-gatsby/gatsby-config.js]:
```suggestion
        cookieDomain: 'tealgarden.deora.earth',
```

**Review by johba37 — CHANGES_REQUESTED** [review]
all good, except cookie domain

File: packages/landing-gatsby/gatsby-config.js (hunk 1 above)
Comment (simonweniger) [reply packages/landing-gatsby/gatsby-config.js:28]: we currently dont use google optimize. I wanted to keep the pace holder, that we can easy add it when we use google optimize in the future.

File: packages/landing-gatsby/gatsby-config.js (hunk 2 above)
Comment (simonweniger) [reply packages/landing-gatsby/gatsby-config.js]: thanks, i forgot that

**Pull request merged**